WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/IssueRefundViewController.xib

```diff
@@ -68,11 +69,11 @@
             <rect key="frame" x="0.0" y="0.0" width="417" height="71"/>
             <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
             <subviews>
-                <stackView opaque="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Tz4-uY-TGt">
+                <stackView opaque="NO" contentMode="scaleToFill" spacing="8" translatesAutoresizingMaskIntoConstraints="NO" id="Tz4-uY-TGt">
```

Comment:
nit: could 8px spacing also be specified in `IssueRefundViewController.configureHeaderStackView` so that it's clear what properties we override?

Reply:
Sure!